libs/langchain/langchain/llms/replicate.py

```diff
@@ -46,6 +46,14 @@ class Config:
 
         extra = Extra.forbid
 
+    @property
+    def lc_secrets(self) -> Dict[str, str]:
+        return {"replicate_api_token": "REPLICATE_API_TOKEN"}
```

Comment:
Note: It seems like LangchainJS uses `REPLICATE_API_KEY` instead of `REPLICATE_API_TOKEN`. Haven't changed that as this would be a breaking change I assume?

Reply:
yea would be breaking. if we wanted could unify the secrets (to be TOKEN or API_KEY, no strong preference) and update the constructor (of whichever class we change) to accept both params/env vars, and just log a deprecation warning if they use the old one

Reply:
can also do that in a later PR